File: doc-framework/test/templates/without-connect.tpl.go
package main

import (
	"fmt"
	"encoding/json"

	"github.com/kuzzleio/sdk-go/protocol/websocket"
	"github.com/kuzzleio/sdk-go/kuzzle"
)

func main() {
	c := websocket.NewWebSocket("kuzzle", nil)
	kuzzle, _ := kuzzle.NewKuzzle(c, nil)

	[snippet-code]

	fmt.Println("Success")
}

File: doc-framework/test/templates/without-connect.tpl.java
import io.kuzzle.sdk.*;

public class CodeExampleGenericClass {
    private static Kuzzle kuzzle;

    public static void main(String[] args) {
      WebSocket ws = new WebSocket("kuzzle");
      kuzzle = new Kuzzle(ws);
      [snippet-code]

      System.out.println("Success");
    }

}
